__tests__/app.test.js

```diff
@@ -0,0 +1,34 @@
+const app = require('../app');
+const seed = require('../db/seeds/seed');
+const data = require('../db/data/test-data/index');
+const connection = require('../db/connection');
+const request = require('supertest');
+const { forEach } = require('../db/data/test-data/articles');
+
+afterAll(() => {
+    return connection.end();
+})
+
+beforeEach(() => {
+    return seed(data);
+})
+
+describe("/api/topics", () => {
+    test("200: response with a status of 200", () => {
+        return request(app).get("/api/topics").expect(200)
+    });
```

Comment:
This is a good first test in terms of TDD because it starts simple and builds up